vizro-core/docs/index.md

```diff
@@ -1,3 +1,285 @@
 # Vizro
 
 Vizro is a toolkit for creating modular data visualisation applications
+
+#### Quickstart
+<div class="card-section-wrapper" style="display: block;">
+<div class="responsive-grid">
+<a class="card-wrapper" href="/vizro/pages/tutorials/first_dashboard/">
+  <div class="card">
+    <div class="card-content">
+      <h5>Get started</h5>
+      <p>
+        Install Vizro and get started with your first dashboard.
+      </p>
+    </div>
+  </div>
+</a>
+
+<a class="card-wrapper" href="/vizro/pages/tutorials/explore_components/">
+  <div class="card">
+    <div class="card-content">
+      <h5>Tutorial</h5>
+      <p>
+        Follow our step-by-step tutorial to learn about all the available features.
+      </p>
+    </div>
+  </div>
+</a>
+
+</div>
+</div>
+<br>
+
+#### User Guide
```

Comment:
Would scrap all the indivudual user guides, and rather explain what user guides are, and link with a card to the main section.

Reply:
Hmm.. I actually kind of like that everything is on one page, it gives you a quick overview of what the package enables without having to click through anything.

 @Joseph-Perkins - do we want cards for each individual section or just the main ones? If we go for the main ones, we would have 5 cards in total for the homepage. It would also be okay for me, but before I remove all of the cards, let's align on this first?

Reply:
I find it overwhelming and not focussed. I think the landing page should provide an orientation of where I can find an answer, rather than providing a link to every single section (which will be also a nightmare to maintain 🙈 )

Reply:
Yeah, I definitely get your point. Saw a few examples of this as well - either way, if the two of you prefer that, I am happy to change it. But would like to align on this first before I change it everywhere 😄

Reply:
Talked with @Anna-Xiong and she also preferred having less cards visually. So majority wins 😄 I like the outcome though. 

Would leave it at 4 cards though:
![Screenshot 2023-09-13 at 14 56 21](https://github.com/mckinsey/vizro/assets/90609403/73036042-4c73-45c6-b348-024d08811473)